target_postgres/json_schema.py

```diff
@@ -4,6 +4,7 @@
 from jsonschema import Draft4Validator
 from jsonschema.exceptions import SchemaError
 
+NULL = 'null'
```

Comment:
Should we create one for each JSON Schema type we use?

Reply:
Prolly. I was doing the minimal change first. I think there's reason to do this regardless though